### ISSUE TITLE
Meta Quest 3 Companion App shows JKA only but won't launch it

### ISSUE THREAD

**Issue by itguy327** [DrBeef/JKXR#25]
Meta Quest 3 Companion App shows JKA only but won't launch it. I have assets for both Jedi Outcast and Academy loaded. I can play Jedi Outcast. I open the Companion app and see JKA is listed but I can't launch it, it only launches Jedi Outcast. Jedi Outcast doesn't show up in the companion app at all.

Thank you, the taste I got was amazing and I'm eager for more

**Comment by baggyg** [comment]
Have you got a screenshot of both folders and where you copied the assets? Guy recently had JK3 in JK2. We can usually diagnose from seeing it

**Comment by itguy327** [comment]
> Have you got a screenshot of both folders and where you copied the assets? Guy recently had JK3 in JK2. We can usually diagnose from seeing it

I followed these instructions. I honestly think I'm just being dense about the UI launch thing.

Thank you 

https://trello.com/c/C0YTFpvX/48-quest-quest-2-installation-instructions

![JK2](https://github.com/user-attachments/assets/3ec442e3-f917-4914-a84c-37fee2b7362a)
![JK3](https://github.com/user-attachments/assets/59a71a4d-9b20-46b9-babf-5ce11b4fa2aa)

**Comment by baggyg** [comment]
Those look fine to me. Companion app should recognise Jedi Academy. 

Perhaps you are misunderstanding how to select that. What do you have in the Companion App selected?

**Comment by itguy327** [comment]
> Those look fine to me. Companion app should recognise Jedi Academy.
> 
> Perhaps you are misunderstanding how to select that. What do you have in the Companion App selected?

Companion app does show Jedi Academy, I think I am just no understanding how to start it. 

secondary question: should the companion app also show JKO?

**Comment by baggyg** [comment]
In this dropdown it should show both Jedi Outcast and Jedi Academy. Click on the one to launch and then click the launch button below 
![image](https://github.com/user-attachments/assets/cf0f1371-6eff-40d7-a082-5bb687b8575a)

You can also see in this section below what it is actually trying to do:

![image](https://github.com/user-attachments/assets/a26bf08d-eff0-4868-9a08-d44237894b26)

**Comment by itguy327** [comment]
> In this dropdown it should show both Jedi Outcast and Jedi Academy. Click on the one to launch and then click the launch button below ![image](https://private-user-images.githubusercontent.com/4569081/396122266-cf0f1371-6eff-40d7-a082-5bb687b8575a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQzNTc3ODYsIm5iZiI6MTczNDM1NzQ4NiwicGF0aCI6Ii80NTY5MDgxLzM5NjEyMjI2Ni1jZjBmMTM3MS02ZWZmLTQwZDctYTA4Mi01YmI2ODdiODU3NWEucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTZUMTM1ODA2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9ZDFmNTE2NDg0ZmMyZDZmNTdkNjUyNGI2NWUzYjU2Mjc5ODYzMTEzYWI3YzhkZDZhNmI2NmRkOTVlMWUxNjQxYiZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.xmzTnPaa-nOOvh7ovqysiye6M0eCUz3kIHdPP8bLk80)
> 
> You can also see in this section below what it is actually trying to do:
> 
> ![image](https://private-user-images.githubusercontent.com/4569081/396122443-a26bf08d-eff0-4868-9a08-d44237894b26.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzQzNTc3ODYsIm5iZiI6MTczNDM1NzQ4NiwicGF0aCI6Ii80NTY5MDgxLzM5NjEyMjQ0My1hMjZiZjA4ZC1lZmYwLTQ4NjgtOWEwOC1kNDQyMzc4OTRiMjYucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTIxNiUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDEyMTZUMTM1ODA2WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9NDg4MmQ3NjdiOTM2NjNjYjFjMDdhY2I2NDYxMWE1ZjZmM2ZjM2M5YWM4NjY4NGM4MTRkZDBlMjhiM2RkZjIxZCZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.xOwzUvDUWlTk5fiJibNmKXakRfptI7VQ8_hszSmloEI)

Maybe it is not rendering correctly for me but I don't see a drop down in the typical sense. I do see an icon for Jedi Academy, which I click and then click Launch but it still brings up the JO menu.

Again, I'm likely being super daft. I'll try to get a screenshot if I can't progress.

Thank you again

**Comment by baggyg** [comment]
Yes, really need a screenshot of whats selected and what it says in the commandline area on the 2nd image I sent

**Comment by itguy327** [comment]
> Yes, really need a screenshot of whats selected and what it says in the commandline area on the 2nd image I sent

Just read something in the instructions. It indicates Patreon required for JKA. Is that accurate. Just making sure I didn't miss a step.

**Comment by baggyg** [comment]
No, thats outdated. All versions are now public
